src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -25,7 +25,10 @@ object Run  {
 
   def apply(pipeline: Pipeline): Run = {
     val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
-    val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}]"
+    def tag: String = {
+      val shard = pipeline.key.index.map(x => s":$x").getOrElse("")
+      s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}$shard]"
+    }
```

Comment:
In this specific case w/ the current state of things I don't have a strong opinion either way. Call it a slight preference towards miguel's thing but not enough that I'd have noticed if I was reviewing w/o bias